spec/mono/1.0.md

```diff
@@ -0,0 +1,61 @@
+<!-- draft -->
+# `vt6/mono1.0` - Monospace layouting on terminals
+
+The canonical URL for this document is <https://vt6.io/std/mono/1.0/>.
+
+**This is a non-normative draft.**
+
+The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "SHOULD NOT", "RECOMMENDED",  "MAY", and "OPTIONAL" in this document are to be interpreted as described in [RFC 2119](https://tools.ietf.org/html/rfc2119).
+
+This document uses the predefined property types from [section 2.4 of `vt6/core1.0`](https://vt6.io/std/core/1.0/#section-2-4).
+
+## 1. Scope
+
+This module extends the notion of terminals introduced in [`vt6/term`](https://vt6.io/std/term/) with additional definitions and assurances for text layouting using monospace fonts.
+Unless the `mono.enabled` property is set to true, clients MUST NOT assume that texts are rendered with monospace fonts.
+Clients that use strings of equal length and box-drawing characters to produce a layout may find that the actual rendering does not match their expectation.
+
+*Rationale:* The idea of character-grid layouts is fundamentally incompatible with wide Unicode support.
+Letters from non-Latin scripts rarely fit into grid cells whose size is optimized for Latin letters.
+Furthermore, the existence of combining characters and grapheme clusters means that applications cannot depend on the equivalence of "one character = one grid cell" when calculating the width that a certain string occupies on the character grid.
+That's why monospace layouting is not enabled by default.
+In fact, VT6-compliant terminals may not offer monospace layouting at all.
+
+### 1.1. Dependencies to other modules
+
+VT6 servers MUST NOT agree to using this module unless they have already agreed to using any version of `vt6/term` on the same server connection.
+
+### 1.2. Monospace layouting
+
+We take the term **terminal** to refer to a terminal as defined in [`vt6/term1.0`](https://vt6.io/std/term/1.0/).
+When the value of the `mono.enabled` property is changed, the server SHALL append a paragraph separator (U+2029) to the terminal document, and move the terminal's cursor to point at the end of the terminal document, after that paragraph separator.
+After this operation, the terminal's cursor SHALL NOT move past this paragraph separator, even when the specification of another module would allow it.
```

Comment:
What about text selection by mouse? E. g. when I want to reuse a text snippet of non-mono output again in a new mono-'environment'? Does mouse selection not imply setting the cursor?

Reply:
This specification does not concern a cursor operated by the user. The cursor we're talking about here is denoting the place where additional stdout gets inserted into the terminal document.